src/lab3/structures/renderers/ImageRenderer.ts

```diff
@@ -45,6 +46,16 @@ export default abstract class ImageRenderer extends CommonRenderer {
 
         pixelsStream.push({ r: color, g: color, b: color });
       },
+      onRenderStart: () => {
+        console.log('Rendering started');
+        this.linesRendered = 0;
+      },
+      onRowEnd: () => {
+        this.linesRendered++;
+        if (this.linesRendered % 10 === 0) {
+          console.log(`Rendered ${this.linesRendered} lines`);
+        }
+      },
```

Comment:
Why do we need this logging here for every 10 rows? Is this helpful for debugging?